fingertip/plugins/reboot.py

```diff
@@ -0,0 +1,18 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2024 Red Hat, Inc., see CONTRIBUTORS.
+
+def main(m):
+    with m:
```

Comment:
too defensive for fingertip =) just fire-and-forget, worst case it'd get snapshotted mid-boot